static/css/style.css

```diff
@@ -849,18 +851,27 @@ fieldset[disabled] .source-buttons .btn-primary.active {
   border: 1px solid transparent;
   white-space: nowrap;
   line-height: 1.42857143;
-  border-radius: 0px;
   -webkit-user-select: none;
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
   text-transform: uppercase;
   text-shadow: none;
-  letter-spacing: .15em;
+  letter-spacing: .05em;
   font-weight: bold;
-  font-size: .9em;
```

Comment:
Yeah, it does have an effect, although they are still different. I've reverted it, since I'm not sure why @tstrachota did that. There's a few other changes in that class, do you want them all reverting?